controllers/toolchainstatus/toolchainstatus_controller_test.go

```diff
@@ -656,7 +657,7 @@ func TestToolchainStatusConditions(t *testing.T) {
 				HasHostOperatorStatus(hostOperatorStatusReady()).
 				HasMemberClusterStatus(memberCluster("member-1", ready()), memberCluster("member-2", ready())).
 				HasRegistrationServiceStatus(registrationServiceReady(conditionReady(toolchainv1alpha1.ToolchainStatusRegServiceReadyReason), conditionReadyWithMessage(toolchainv1alpha1.ToolchainStatusDeploymentRevisionCheckDisabledReason, "access token key is not provided"))).
-				HasHostRoutesStatus("http://api-toolchain-host-operator.host-cluster/api", hostRoutesAvailable())
+				HasHostRoutesStatus("http://api-toolchain-host-operator.apps.host-cluster/api", hostRoutesAvailable())
```

Comment:
just curious , is there a reason for not using `hostProxyURL` here? 
```suggestion
				HasHostRoutesStatus(hostProxyURL, hostRoutesAvailable())
```

Reply:
We are specifying the route spec. at L645, so that changes the host Proxy URL to be appended with api. Hence I kept it the way it was

Reply:
Oh my bad, I missed that is expecting `http` (and not `https`) and the `/api` endpoint. Sorry for that 👍